cmd/scw/main.go

```diff
@@ -74,6 +74,7 @@ func main() {
 		Stdout:    colorable.NewColorableStdout(),
 		Stderr:    colorable.NewColorableStderr(),
 		Stdin:     os.Stdin,
+		BetaMode:  BetaMode,
```

Comment:
Could you add documentation about how the beta flag is handled? Is it with an environment variable? or is it a flag passed in the CLI?